recipes/full_finetune_single_device.py

```diff
@@ -150,6 +150,9 @@ def setup(self, cfg: DictConfig) -> None:
         """
         self._metric_logger = config.instantiate(cfg.metric_logger)
 
+        # log config with parameter override
+        self._metric_logger.log_config(cfg)
```

Comment:
Do we plan to add this integration to our other recipes as well or just this one for now?

Reply:
(Separate q: @RdoubleA can we easily support this for our other metric loggers as well?)

Reply:
yes - we can leave that as a follow up